test/AutoRest.TestServerLowLevel.Tests/body-array.cs

```diff
@@ -11,6 +11,8 @@
 using System.Xml;
 using AutoRest.TestServer.Tests.Infrastructure;
 using Azure.Core;
+using Azure;
```

Comment:
The PR is to update comments. Why is the Test  updated?

Reply:
We manually compose some of the generated examples (with some modification) in the test, to make sure it can compile.